resume-buddy/src/test/java/com/google/sps/CommentsTest.java

```diff
@@ -0,0 +1,96 @@
+package com.google.sps;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import com.google.sps.data.Comment;
+import com.google.sps.servlets.CommentServlet;
+import java.text.ParseException;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public class CommentsTest {
+  private DatastoreService datastore;
+  private Entity MATCH;
+  private Entity COMMENT_1;
+  private Entity COMMENT_2;
+  private Entity COMMENT_3;
```

Comment:
Non-static private variables should be lowercase with camelcase: `match, comment1, comment2, comment3`.